eb.tf

```diff
@@ -706,3 +706,9 @@ resource "aws_elastic_beanstalk_environment" "env" {
   }
 }
 
+data "aws_elastic_beanstalk_solution_stack" "solution_stack" {
+  count       = var.solutions_stack_name_regex ? 1 : 0
```

Comment:
`solutions_stack_name_regex` is a string, it's better to do a conditional based on its length or value (i.e. `try(var.solutions_stack_name_regex, "") != "" ? 1 : 0`.

Reply:
Nice one @luiz-dnx , i didn't saw that.
Fixed now.